verification/curator-service/api/test/model/source.test.ts

```diff
@@ -1,20 +1,21 @@
 import { Source } from '../../src/model/source';
+import fullSource from './data/source.full.json';
+import minimalSource from './data/source.minimal.json';
 
```

Comment:
maybe a test for Field to demonstrate you can't specify a Field without a regex? same maybe for Parser and awsLambdaArn, and Schedule and awsRuleArn?

Reply:
Yeah, we should; I was keeping this at the Source level, for now. WDYT about moving Origin/Automation to separate files, and adding detailed tests for each?

Reply:
sgtm!